app/javascript/topNavigation/__tests__/utilities.test.js

```diff
@@ -41,7 +40,7 @@ describe('top navigation utilitities', () => {
   describe('initializeMobileMenu', () => {
```

Comment:
Is there anything tested in this file that isn't tested in the [Cypress spec](https://github.com/forem/forem/pull/15653/files#diff-db5c1c2c14d826a903a7b9655f09d0775d7dbb2e7d63739553cb459cdbb9768d)? I think we can definitely remove the open/close on click tests since they're covered in Cypress, but I'm not sure about the others.

Reply:
I honestly have no idea - you do know my lack of experience when it comes to tests... I just removed/updated things that didn't really make any sense due to some changes I've made in frontend.

^ this is my gently asking for help here :D

Reply:
I'll take a look, maybe we can get rid of a bunch 🧹

Reply:
Done - added a couple of assertions to Cypress and got rid of the member nav menu tests from this file since we already had a lot duplicated in Cypress. The others could be ported to Cypress too but adding them new doesn't seem like part of this PR 🙂

Reply:
I think it broke the build..? :D

Reply:
Ummm WOOPS - let me take a look!

Reply:
it's weird - keeps failing on an assertion that passes fine locally... anyway, it was a "while I'm here" addition, and we're not missing anything if I just take it out for now. Hopefully should go green when it rebuilds 🙏

Reply:
hahaha, it went red :D